ReSharper.FSharp/src/FSharp.Common/src/Checker/FcsCheckerService.fs

```diff
@@ -25,6 +25,34 @@ open JetBrains.ReSharper.Psi.Tree
 open JetBrains.ReSharper.Psi.VB
 open JetBrains.Util
 
+[<AutoOpen>]
+module FcsProjectProviderUtils =
```

Comment:
Why this module was moved to `FcsCheckerService.fs`?

Reply:
I needed the `getModuleProject` function, so I moved the module containing it closer to IFcsProjectProvider

Reply:
What do you think is the best thing to do?